src/yunohost/permission.py

```diff
@@ -35,309 +36,247 @@
 
 logger = getActionLogger('yunohost.user')
 
+SYSTEM_PERMS = ["mail", "xmpp", "stfp"]
 
-def user_permission_list(app=None, permission=None, username=None, group=None):
-    """
-    List permission for specific application
+#
+#
+#  The followings are the methods exposed through the "yunohost user permission" interface
+#
+#
 
-    Keyword argument:
-        app        -- an application OR sftp, xmpp (metronome), mail
-        permission -- name of the permission ("main" by default)
-        username   -- Username to get informations
-        group      -- Groupname to get informations
 
+def user_permission_list(short=False, full=False, ignore_system_perms=False):
```

Comment:
Are you sure that it's not usefull to be able to filter by permission, username or group ?

Reply:
Hmmm idk, I didn't really need it ... Except maybe the "fetch the info for a specific permission" which could be useful in many places of the code. Dunno if we want to have a `user permission list foo.main` or if we want a `user permission info foo.main`  like we do elsewhere ...

Otherwise if you want to know which permissions are allowed for a specific group/user, you can just run `user group info the_user_or_group`